examples/vectors.js

```diff
@@ -44,6 +46,14 @@ const cervinFeature = new ol.Feature({
 cervinFeature.getGeometry().set('altitudeMode', 'clampToGround');
 
 
+const modelFeatures = [-1, -1 / 2, 0, 1 / 2, 1, 3 / 2].map(
+    factor => new ol.Feature({
+      geometry: new ol.geom.Point([852641, 5776749, 4500]),
+      'rotation': factor * Math.PI
```

Comment:
remove quotes?

Reply:
I like to be ready for GCC compilation of the examples :P